de.metas.business/src/main/java/de/metas/uom/IUOMConversionBL.java

```diff
@@ -133,8 +133,16 @@ BigDecimal convertQty(
 	 *
 	 * @return converted qty (std precision)
 	 */
+	@Deprecated
 	Optional<BigDecimal> convert(I_C_UOM uomFrom, I_C_UOM uomTo, BigDecimal qty);
 
+	/**
+	 * Convert qty to target UOM and round.
+	 *
+	 * @return converted qty (std precision)
+	 */
+	Optional<BigDecimal> convert(@NonNull final UomId fromUomId, @NonNull final UomId toUomId, @NonNull final BigDecimal qty);
```

Comment:
it bothers me quite a lot that we have `Quantity.of()`, but then we have `Quantitys.of()` instead of `Quantities.of()`.

Reply:
This will make my implementation way more ugly imo.
Compare
```java
.heightInCM(iuomConversionBL.convertQtyTo(Quantitys.create(packingMaterial.getHeight(), fromUomId), toUomId).orElse(Quantitys.create(1, toUomId)).toBigDecimal().intValue())
```
with 
```java
.heightInCM(iuomConversionBL.convert(fromUomId, toUomId, packingMaterial.getHeight()).orElse(BigDecimal.valueOf(1)).intValue())
```

I will add the method, but won't use it here.